Cartfile

```diff
@@ -0,0 +1,18 @@
+###Internal
+
+github "kickstarter/Kickstarter-Prelude" "7a56163ad02d817aeb022f6dcb3820a733d5ec4c"
```

Comment:
Some of these are pinned to commit hashes, that's just to ensure that they're unchanged from before for those dependencies that had submodules referencing non-release/tagged commits.